pkg/network/server.go

```diff
@@ -946,27 +940,26 @@ func (s *Server) handleExtensibleCmd(e *payload.Extensible) error {
 	if !ok { // payload is already in cache
 		return nil
 	}
-	switch e.Category {
-	case consensus.Category:
-		if s.consensus != nil {
-			s.consensus.OnPayload(e)
-		}
-	case stateroot.Category:
-		err := s.stateRoot.OnPayload(e)
+	handler := s.extensHandlers[e.Category]
```

Comment:
This is explicitly mentioned in the commit message, C# node doesn't do that and it has very little value. Implementation-wise it means keeping a registry of "good" categories in `payload` package, rather than keeping them in the services (which may have its benefits also (avoiding intersections), but just not the thing we do now).